tests/settings/nersc.yml

```diff
@@ -56,3 +56,23 @@ system:
           qos: bigmem
           cluster: escori
           description: large memory jobs
+
+  perlmutter:
+    hostnames: ["login*"]
+    moduletool: lmod
+    compilers:
+      enable_prgenv: true
+      prgenv_modules:
+        gcc: PrgEnv-gnu
+        cray: PrgEnv-cray
+        nvhpc: PrgEnv-nvidia
```

Comment:
lets change this to `PrgEnv-nvhpc` i think we will start using this going forward.

Reply:
Done !